examples/sso-advanced.html

```diff
@@ -431,45 +524,46 @@
 }
 
 function removeIdentity() {
-  var identobj = tizen.sso.authService.getIdentityByJSId(
+  var appIdent = _getAppIdentityByGuiId(
     document.getElementById('ident_options').value);
-  if (identobj == null) {
+  if (appIdent == null) {
     _logData('identity NOT selected/found. Please select/create an identity first');
     return;
   }
-  _logData('identity found with JSId: ' + identobj.jsid);
-  identobj.remove().then(onIdentityRemoved,
+  _logData('identity to be removed: ' + appIdent.guiId);
+  appIdent.identityObject.removeIdentity().then(onIdentityRemoved,
     function(err) {_logData('remove failed: ' + err);});
 }
 
 function signout() {
-  var identobj = tizen.sso.authService.getIdentityByJSId(
+  var appIdent = _getAppIdentityByGuiId(
     document.getElementById('ident_options').value);
-  if (identobj == null) {
+  if (appIdent == null) {
     _logData('identity NOT selected/found. Please select/create an identity first');
     return;
   }
-  _logData('identity found with JSId: ' + identobj.jsid);
-  identobj.signout().then(function(msg) {
+  _logData('identity to be signeout: ' + appIdent.guiId);
+  appIdent.identityObject.signout().then(function(msg) {
     _logData('signout succeeded');},
     function(err) {_logData('signout failed: ' + err);});
 }
 
 //authsession interface
 function queryAvailableMechanisms() {
-  var identobj = tizen.sso.authService.getIdentityByJSId(
+  var appIdent = _getAppIdentityByGuiId(
     document.getElementById('ident_options').value);
-  if (identobj == null) {
+  if (appIdent == null) {
     _logData('identity NOT selected/found. Please select/create an identity first');
     return;
   }
-  _logData('identity found with JSId: ' + identobj.jsid);
-  var sessionobj = identobj.getSessionByJSId(document.getElementById('sess_options').value);
-  if (sessionobj == null) {
+  _logData('identity found with GuiId: ' + appIdent.guiId);
+  var appSession = _getAppSessionByGuiId(
+      document.getElementById('sess_options').value);
```

Comment:
squash to one line, we use 100 line limit with gslint, you could check `src/tools/check-coding-style`